.github/workflows/ci.yml

```diff
@@ -118,6 +118,8 @@ jobs:
                   echo "⏳ Waiting 10 seconds to see if it runs"
                   sleep 10
 
+                  cat tmp/plugin-server.log
```

Comment:
This way we have this line once always and also a second time if the plugin server failed somehow (down below) 🤔

Reply:
<Sigh>, Thanks!